jenkins/deployment/community/community-build-cortx.groovy

```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent {
         node {
-            label 'docker-k8-deployment-node'
+            label 'ssc-vm-g4-rhev4-0465.colo.seagate.com'
```

Comment:
Please update label name with some generic one like `community-build-executor`

Reply:
ok. I will update the label.

Reply:
Done